person.rb

```diff
@@ -1,8 +1,14 @@
-class Person
+require_relative 'capitalize'
+require_relative 'nameable'
+require_relative 'decorator'
+require_relative 'trim'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
-  def initialize(age, parent_permission: true, name: 'Unknown')
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    super
```

Comment:
- [ ] When I try to create a new person, this error arises:

![image](https://github.com/danielochuba/oop-school-library/assets/77069011/91a7d304-934b-4ed7-b53b-6655bd4a086c)

This happens because you're calling super with no defined arguments, to solve this, simply use `()` parenthesis.

```suggestion
  def initialize(age, name: 'Unknown', parent_permission: true)
    super()
```
Please consider adding them! 🤓 🔧